docs/reference/flavors/index.md

```diff
@@ -0,0 +1,50 @@
+# Flavors
+
+Any server instance running in {{extra.brand}} Cloud has a **flavor**,
+which defines the number of virtual CPU cores, the amount of virtual
+RAM, and other performance-related factors.
+
+
+## Naming convention
+
+Flavor names in {{extra.brand}} follow a convention, which can be
+summarized as `X.YcZgb`:
+
+* `X` stands for a lowercase letter identifying the [compute
+  tier](#compute-tiers), with `b` representing the general-purpose
+  tier. It is always followed by a full-stop (`.`).
```

Comment:
Just pointing out here that, in Cleura Cloud Control Panel at least, `X` is an uppercase letter and always equal to `B`.

Reply:
Not anymore. :)